RecoPixelVertexing/PixelTriplets/plugins/gpuPixelDoublets.h

```diff
@@ -75,6 +75,16 @@ namespace gpuPixelDoublets {
     };
 
     constexpr float z0cut = 12.f;
+    constexpr float hardPtCut = 0.5f; 
+    constexpr float minRadius = hardPtCut * 87.f;
```

Comment:
can you comment where `87.f` comes from ?

Reply:
Everybody in CMS should know that a 1 GeV pt track has a radius of 87 cm
(I copied the two line from CA btw)